include/nuttx/tls.h

```diff
@@ -219,6 +219,7 @@ struct tls_info_s
   int16_t tl_cpcount;                  /* Nested cancellation point count */
 #endif
 
+  uint16_t tl_size;                    /* Actual size with alignments */
```

Comment:
but only task can have argc/argv, should tl_size belong to task_info_s

Reply:
If the size is the same for all threads, putting it in `task_info_s` is fine as it is not thread specific. Putting it in-place is also fine as we don't need worry whether in the future if the size is thread-specific.

Reply:
OK